code/API_definitions/device-status.yaml

```diff
@@ -30,9 +31,56 @@ servers:
         description: Base path for the device status API
 tags:
   - name: Device roaming status
-    description: Operations to validate device roaming status and country information
+    description: Operation to get device roaming status and country information (if roaming) synchronously
+  - name: Device status subscription
+    description: Operation to manage subscription on device status event (roaming, connectivity)
+  - name: Notification Callback
+    description: |
+      Notification received on subscription listener side. 
+      
+      **WARNING**: This endpoint must be exposed on the listener side as POST /notifications 
+      will be posted by the Roaming resource server to the url provided by notification 
+      listener.
 paths:
-   /roaming:
+  /notifications:
```

Comment:
shouldn't we put the notification endpoint inside POST /event-notifications as callback?

Reply:
@akoshunyadi - This is a great question. We saw that QoD API went in this direction;

My developer colleagues are not comfortable with this pattern (have the callback in the end point) as it is not really supported by code generator. A good point to discuss in Commonalities?

Reply:
My preference is to have the notification endpoint defined as a callback. However, I understand that code generators can have issues with callbacks, and having the separate notifications path does comply with the design guidelines (though these make it look like the endpoint should be called `eventNotifications` - maybe some editing is needed there). So I will approve this PR so that we can move towards v0.5.0.

Ongoing, I agree this would be a good topic to raise in Commonalities. I have seen three methods for defining notifications discussed:
- As a callback (as we have in QoD API)
- As a separate notifications path, with appropriate documentation (as we have here)
- As a completely separate OAS definition for use by the API consumer notification endpoint

I don't think these methods are incompatible. Why not have all three? But this may also cause issues with code generators if both a callback and separate notification path are defined, so should be discussed in Commonalities.

Reply:
Our proposal with @bigludo7 is to provide the definition as a callback because it documents very well what is  happening (and will align with how it's done in QoD). Also add a separate OAS definition to help implementers of this callback.
As you say, these approaches are not incompatible, so we can benefit from them.
We agree with you to discuss this point in the commonalities to help harmonization between APIs.

Reply:
Thanks @patrice-conil

So is the proposal to modify this PR to include the notification endpoint as a callback? Or will that wait until after the discussion in Commonalities?

Reply:
@eric-murray,
I think it can wait until after discussion in Commonalities. I will propose a new PR after discussion if necessary.

Reply:
Thanks @patrice-conil

Once there are the additional approvals from @monamok and @akoshunyadi, I can merge this.